packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -61,6 +62,16 @@ export class AccountResolver {
       .first();
   }
 
+  //query to return all emails that are in the database
+  @Query((returns) => [EmailType], { nullable: true })
+  public getEmailsLike(@Args('search', { nullable: false }) search: string) {
+    return this.db.emails
+      .query()
+      .select(EMAIL_COLUMNS)
+      .where('email', 'like', `%${search}%`)
```

Comment:
The `%${search}%` will search for that substring anywhere in the email address because it has `%` on both sides. We want our suggestions to be a prefix search so we only need the `%` on the end.